docs/development/testing_configuration_tests.rst

```diff
@@ -20,35 +20,36 @@ Installation
 Running the Config Tests
 ------------------------
 
-In order to run the tests, first create and provision the VM you intend
-to test.
-
-For the staging VMs:
+Testinfra tests are executed against a virtualized staging environment. To
+provision the environment and run the tests, run the following commands:
 
 .. code:: sh
 
     make build-debs
-    make staging
+    make testinfra
 
-The VMs will be set up using either the libvirt or virtualbox Vagrant VM provider,
-depending on your system settings. You'll need to use the appropriate commands below
-based on your choice of provider.
+.. note:: The staging environmment will be destroyed after the tests complete. To keep 
```

Comment:
Ok, if we move to using `verify` instead of `test` then the docs are simplified, at the cost of having to explicitly create the staging env first, which is no big deal IMO.